.gitignore

```diff
@@ -3,4 +3,5 @@
 var/*
 vendor/*
 
+.env
```

Comment:
Why? Does something automatically use this file? You aren't referencing this path anywhere.

Reply:
fixed, removed